src/Jobs/ClearIndexJob.php

```diff
@@ -121,4 +110,34 @@ public function process()
             ));
         }
     }
+
+    public function getBatchOffset(): ?int
+    {
+        return $this->batchOffset;
+    }
+
+    public function getBatchSize(): ?int
+    {
+        return $this->batchSize;
+    }
+
+    public function getIndexName(): ?string
+    {
+        return $this->indexName;
+    }
+
+    protected function setBatchOffset(?int $batchOffset): void
```

Comment:
Similar comment as the properties - can these be private instead of protected? 

Same question applies to the other classes that have protected instead of private